### DISCUSSION TITLE
Image media item should have a core / default alt property

### DISCUSSION THREAD

**Discussion by AaronSadlerUK** [umbraco/Umbraco-CMS#11913] [Features and ideas]
For accessibility and SEO purposes we all know images should have an ALT tag.

So I feel that any image media item should have a default property called alt, which means the alt tag can be managed from a central point within the Media section.

This would make it both easier for editors to manage the image alt tag and in a consistent location.

I don't feel the Name is appropirate for this, as in my experience editors may not associcate Name as ALT text, also the filename which populates this field on upload rarely matches what the alt tag would be

**Comment by nul800sebastiaan** [comment]
FYI: https://github.com/umbraco/Umbraco-CMS/issues/8432

**Reply by RachBreeze** [reply]
Also https://github.com/umbraco/Umbraco-CMS/discussions/10468 :-)

**Comment by sotirisf** [comment]
I've had this issue more than once, especially with multilingual sites. IMHO the ALT property should not only be there, but it should support language variants in some way so that editors are able to add localized ALT strings for each language. That said, there's always the scenario where an image is used in multiple places and the ALT property needs to be generated according to context - in this case, no field can do the job and it's usually done programmatically. But for plain old ALT tags, a multilingual ALT field would really help.
